recipe/meta.yaml

```diff
@@ -37,11 +36,12 @@ test:
   source_files:
     - tests
   commands:
-    - pytest tests
+  # Skipping test_session_with_selfsigned_ca because pytest-httpserver is not on defaults
+    - pytest tests -k "not test_session_with_selfsigned_ca"
+    - pip check
```

Comment:
You may want to do `pip check` first. Problems in the dependencies should be caught before unit tests are performed.